### DISCUSSION TITLE
Why do I get two results?

### DISCUSSION THREAD

**Discussion by elchupacabrinski** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#245] [Q&A]
Whenever I'm doing IMG2IMG upscaling with this extension, I get 2 results.

One that's changed much and one that looks like it was upscaled with Topaz Gigapixel or the like. Why is that and what's the second one for?

Thanks :)

**Comment by AugmentedRealityCat** [comment]
I can observe something similar when I use ControlNet at the same time as Multi-diffusion - ControlNet basically appends the resulting picture with any control image you had been feeding it through the preprocessors. 

Was this your case ?
